pom.xml

```diff
@@ -138,7 +138,18 @@ The MIT License (MIT)
             <configuration>
               <excludes combine.children="append">
                 <exclude>checkstyle:/src/site/resources/.*</exclude>
-                <exclude>findbugs:.*</exclude>
+                <exclude>findbugs:org.cactoos.text.TextEnvelopeTest</exclude>
```

Comment:
@ilyakharlamov Is there anything we can do in the future to remove these exclusions? If yes, WDYT put a PDD what it need to do?

Reply:
@fabriciofx One issue can be fixed if #1028 is merged. I also opened a PR for qulice that will allow exclusions on per-class, per-rule and per-method basis.

Reply:
@ilyakharlamov Right.